package.json

```diff
@@ -58,15 +59,15 @@
     "eslint-plugin-vue": "8",
     "growl": "^1.10.0",
     "https-proxy-agent": "^2.2.0",
-    "jest": "24",
```

Comment:
I am not sure, but if you remove `jest` package, then shouldn't you also remove `babel-jest`?